fms/utils/serialization.py

```diff
@@ -98,7 +109,7 @@ def load_state_dict(
     initial_device: where the state dict will be loaded. if meta, return None.
     """
     if model_path is None or initial_device.type == "meta":
-        return None
+        return {}
```

Comment:
Will this cause issues? Are we handling the dict?

Reply:
yes in most cases the sd was a {} not None, I think this is a case where the mypy checks were finding problematic code. I don't think the `None` here broke anythiing, but it had the potential to cause issues.